lib/livebook/notebook.ex

```diff
@@ -101,6 +101,57 @@ defmodule Livebook.Notebook do
     put_in(notebook.setup_section.cells, [%{cell | id: Cell.setup_cell_id()}])
   end
 
+  @doc """
+  TODO
+  """
+  @spec move_output_to_canvas(t(), Cell.t(), Section.t()) :: t()
+  def move_output_to_canvas(notebook, cell, section) do
+    notebook
+    |> update_in(
+      [
+        Access.key(:sections),
+        access_by_id(section.id),
+        Access.key(:cells),
+        access_by_id(cell.id)
+      ],
+      fn cell ->
+        %{cell | output_location: %{x: 0, y: 0, w: 4, h: 2}}
+      end
+    )
+  end
+
+  @doc """
+  TODO
+  """
+  @spec move_output_to_notebook(t(), Cell.t(), Section.t()) :: t()
+  def move_output_to_notebook(notebook, cell, section) do
+    notebook
+    |> update_in(
+      [
+        Access.key(:sections),
+        access_by_id(section.id),
+        Access.key(:cells),
+        access_by_id(cell.id)
+      ],
+      fn cell ->
+        %{cell | output_location: nil}
```

Comment:
maybe add another variable to the cell, so when moving the output back to the notebook, the location gets saved.
I think it is more expected behaviour when the output moves back to the same location

Reply:
I think it's fine, moving output back and forth is not likely and when they remove an output it's likely that the user rearranges things to fill the empty space :)